docs/source/services/timelock_service/migration.rst

```diff
@@ -0,0 +1,110 @@
+.. _timelock-migration:
+
+Migration to External Timelock Services
+=======================================
+
+.. danger::
+
+   Improperly configuring one's cluster to use external timestamp and lock services can result in **SEVERE DATA
+   CORRUPTION**! Please contact the AtlasDB team before attempting a migration.
+
+Why Migration?
+--------------
+
+AtlasDB assumes that timestamps returned by the timestamp service are monotonically increasing. In order to preserve
+this guarantee when moving from an embedded timestamp service to an external timestamp service, we need to ensure
+that any timestamps the external timestamp service issues must be larger than any timestamps that the embedded
+service issues. Otherwise, this can lead to serious data corruption.
+
+The migration process must be run offline (that is, with no AtlasDB clients running during migration) and basically
+consists of the following steps:
+
+1. Set up the external timestamp service.
+2. Obtain a timestamp TS that is at least as large as the largest timestamp issued.
```

Comment:
Ah, I get it! "TS" is referred to below, and you also dig into how to get the largest timestamp issued. We could format like "Obtain a timestamp `TS` that is at least..." to make it clearer that it's a variable you're using.